config/i18n-tasks.yml

```diff
@@ -0,0 +1,143 @@
+# i18n-tasks finds and manages missing and unused translations: https://github.com/glebm/i18n-tasks
```

Comment:
Let's add the `bundle exec i18n-tasks health` task to the PR checklist and CI.

Reply:
Also, run `bundle exec i18n-tasks normalize` to clean up the translation file and commit.

Reply:
> Let's add the bundle exec i18n-tasks health task to the PR checklist and CI. @joemasilotti - I guess I need to move this gem to developemnt and test group then ?

Reply:
Ah, yeah, that's correct.